Ref/feature/panel size

TheSintoras opened the pull request:
Panel Size is now a setting instead of saving into its own file.

cblech commented on Assets/Scripts/System/WorkspaceSaveSystem.cs:
```diff
@@ -2,31 +2,41 @@
 using System;
 using System.IO;
+using Zenject;
+using static Assets.Scripts.System.SettingsSystem;
```
`using System.IO;` and `using static Assets.Scripts.System.SettingsSystem;` are not needed

cblech commented on Assets/Scripts/System/WorkspaceSaveSystem.cs:
```diff
@@ -2,31 +2,41 @@
 
 namespace Assets.Scripts.System
 {
+
```
Code style: Don't put an empty line after opening a code block, i.e. after an `{`

cblech commented on UserSettings/EditorUserSettings.asset:
```diff
@@ -6,31 +6,31 @@ EditorUserSettings:
   m_ConfigSettings:
     RecentlyUsedScenePath-0:
-      value: 22424703114646680e0b0227036c6c111b07142f1f2b233e2867083debf42d
+      value: 224247031146466e0307183b370f6b365932002b21382a353e662e30ebc63d10e7eb3becf30e3a2f2d31ec390a3b4f2afc031d12
```
Please reset changes to this file. It's not needed in this pull request

TheSintoras replied:
Done

cblech commented on Assets/Scripts/System/WorkspaceSaveSystem.cs:
```diff
@@ -2,31 +2,41 @@
         public void Load(DynamicPanelsCanvas canvas)
         {
-            if(!File.Exists(saveFilePath))
+            if(settingSystem.panelSize.Get()=="")
```
Code style: please add some spaces between "commands". 
`if (settingSystem.panelSize.Get() == "")`
Hint: use auto formatting to fix these simple code style errors before committing

TheSintoras committed:
PanelSize is now saved in a setting

Was previously in its own file.
Now uses the StringUserSetting from SystemSettings

cblech committed:
Fixed formatting
